integrated_channels/integrated_channel/models.py

```diff
@@ -62,3 +77,55 @@ class EnterpriseCustomerPluginConfiguration(TimeStampedModel):
 
     class Meta:
         abstract = True
+
+    def transmit_learner_data(self):
+        """
+        Collect and transmit learner data for the enterprise_customer.
+        """
+        for learner_data in self.collect_learner_data():
+            LOGGER.info(learner_data)
+
+            # TODO ENT-221 will handle transmitting the learner data to the channel
```

Comment:
FYI, I had intended the logic for sending data to a channel to live in the transmitters folder of each integrated_channel, so if you want to create a stub method to call that could be a place to put it.